minesweepr.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Minesweepr Game</title>
+  <style>
+  table, td {
+    background: #DCDCDC;
+    border: 1px solid #808080;
+    border-collapse: collapse;
+    text-align: center;
+  }
+
+  td {
+    height: 20px;
+    width: 20px;
+  }
+  </style>
+</head>
+
+<body>
+  <table>
```

Comment:
Yes, that's the idea. A table (as opposed to using divs or another block element) will maintain its row length even if the window is too small to display all of it.